examples/runners.jl

```diff
@@ -0,0 +1,87 @@
+# # Runners
+# ```@raw html
+# <video width="auto" controls autoplay loop>
+# <source src="../runners.mp4" type="video/mp4">
+# </video>
+# ```
+#
+# Let's consider a race to the top of a mountain. Runners have been scattered about
+# a map in some low lying areas and need to find the best path up to the peak.
+#
+# We'll use the [`AStar`](@ref) pathfinder and a [`HeightMap`](@ref) to simulate this.
+
+# ## Setup
+using Agents
+using Random
+using FileIO # To load images you also need ImageMagick available to your project
+
+@agent Runner GridAgent{2} begin end
+
+# Our agent, as you can see, is very simple. Just an `id` and `pos`ition provided by
+# [`@agent`](@ref). The rest of the dynamics of this example will be provided by the model.
+
+function initialize(map_url; goal = (128, 409), seed = 88)
+    ## Load an image file and convert it do a simple representation of height
+    heightmap = floor.(Int, convert.(Float64, load(download(map_url))) * 255)
+    ## The space of the model can be obtained directly from the image.
+    ## Our example file is (400, 500).
+
+    ## The pathfinder. We use the [`Chebyshev`](@ref) metric since we want the runners
+    ## to look for the easiest path to run, not just the most direct.
```

Comment:
The word "Chebyshev" here does not make sense. We are talking about the possible directions of travel (4 versus 8), not a way to measure distances in 2D space.

Reply:
No, we are definitely talking about a way to measure distances in 2D space, since this is a discussion about the `cost_metric`. This doesn't concern the `moore` property (which in 2D is the 4 vs 8 question).

Reply:
Alright, I'm confused then. What does "to look for the easiest path to run, not just the most direct" mean? Specifcally what does "direct" mean?

Reply:
So I agree it's probably not as clear as it should be. What I'm trying to say is that using the default metric does not penalize height (without providing a custom weight). 

With `pathfinder = (cost_metric = HeightMap(heightmap),)`:

https://user-images.githubusercontent.com/3059508/109482860-d43a7180-7a7e-11eb-8e1a-191481a30c80.mp4

See how they basically run directly to the top in comparison?

Reply:
That's interesting. Should `Chebyshev` be made the default in `HeightMap` then?

Reply:
That may be useful. But I suspect I'm missing something in George's argument still. He's usually right on these things :P Best to wait for him to confer.